anxcloud/schema_dns_record.go

```diff
@@ -0,0 +1,72 @@
+package anxcloud
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func schemaDNSRecords() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"zone_name": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "The parent zone",
+		},
+		"name": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "TBD",
+		},
+		"data": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "TBD",
+		},
+		"type": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "TBD",
+		},
+		"records": {
+			Type:        schema.TypeList,
+			Computed:    true,
+			Description: "List of known records on the zone",
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"identifier": {
+						Type:        schema.TypeString,
+						Computed:    true,
+						Description: "The record's identifier",
+					},
+					"type": {
+						Type:        schema.TypeString,
+						Computed:    true,
+						Description: "The DNS record type",
+					},
+					"rdata": {
+						Type:        schema.TypeString,
+						Computed:    true,
+						Description: "The DNS record data",
+					},
+					"name": {
+						Type:        schema.TypeString,
+						Computed:    true,
+						Description: "The name of the DNS record",
+					},
+					"region": {
+						Type:        schema.TypeString,
+						Computed:    true,
+						Description: "The region for geodns aware records",
+					},
+					"ttl": {
+						Type:        schema.TypeInt,
+						Computed:    true,
+						Description: "The region specific TTL. If null the zone TTL will be used",
```

Comment:
null = `nil` or null = `0`?

Reply:
in `flattenDnsRecords` I can see it's indeed not set, meaning it is `nil` - please change comment to `nil` instead of `null`